indico/modules/events/management/controllers/protection.py

```diff
@@ -85,7 +85,7 @@ def _process(self):
             update_event_protection(self.event, {'protection_mode': form.protection_mode.data,
                                                  'own_no_access_contact': form.own_no_access_contact.data,
                                                  'access_key': form.access_key.data,
-                                                 'visibility': form.visibility.data,
+                                                 'visibility': form.visibility.data if form.visibility else None,
```

Comment:
Should we set it to None or skip this altogether?

FWIW, setting it to None doesn't seem like a problem, the only case where it would end up changing something is when we convert an event from listed to unlisted (not sure if that's possible and/or whether we need this), that had an explicit value set...

Reply:
I wasn't sure either, but this can be confusing if we later enable some kind of option to 'unpublish' an event. (Which I'm not sure is a good idea). So I'll skip it if not present in the form.